src/schema/v2/Feature/FeatureImageType.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLNonNull,
+} from "graphql"
+import { Gravity } from "@artsy/quartz"
+import { scale } from "proportional-scale"
+import { ResolverContext } from "types/graphql"
+import proxy from "schema/v2/image/proxies"
+import { ScaledImage, TScaledImage } from "../image"
+
+export const FeatureImageType = new GraphQLObjectType<
+  Gravity.Feature,
+  ResolverContext
+>({
+  name: "FeatureImage",
+  fields: {
+    url: {
+      type: GraphQLString,
+      args: {
+        version: { type: GraphQLString, defaultValue: "wide" },
+      },
+      resolve: ({ image_urls }, args) => image_urls[args.version],
+    },
+    resized: {
+      type: ScaledImage,
+      args: {
+        width: { type: GraphQLInt },
+        height: { type: GraphQLInt },
+        quality: { type: GraphQLInt, defaultValue: 80 },
+        version: {
+          type: new GraphQLNonNull(GraphQLString),
+          defaultValue: "wide",
+        },
+      },
+      resolve: ({ image_urls, original_height, original_width }, args) => {
+        const url = image_urls[args.version]
+
+        if (
+          !url ||
+          !(original_width && original_height && (args.width || args.height))
+        ) {
+          return null
+        }
+
+        const scaled = scale({
+          width: original_width,
+          height: original_height,
+          maxWidth: args.width!,
+          maxHeight: args.height!,
+        })
+
+        const urls = {
+          "1x": proxy(url, "resize", scaled.width, scaled.height, args.quality),
+          "2x": proxy(
+            url,
+            "resize",
+            scaled.width * 2,
+            scaled.height * 2,
+            args.quality - 25 // Can get away with lower quality for higher DPI
+          ),
+        }
+
+        const image: TScaledImage = {
+          ...scaled,
+          src: urls["1x"],
+          srcSet: `${urls["1x"]} 1x, ${urls["2x"]} 2x`,
+        }
+
+        return image
```

Comment:
I'll extract out and write some specs for this function, which is really the only actual logic in this PR.

Reply:
👌 Yah seems like this can be abstracted away at some point

Reply:
I'll pull this off for now and encode some better information about the model into the default image type just so we can get this merged now. Will follow up though because it is necessary going forward.